### PR TITLE
Update boilerplate and generate CSV through boilerplate scripts

### PR DESCRIPTION
**Pull request by arjunrn**
This PR contains 2 main changes:
1. Update boilerplate
2. Rename the CSV template so that it can be used by boilerplate CSV generation.

### CODE REVIEW

File: config/templates/csv-template.yaml [hunk 1]
```diff
@@ -36,14 +36,7 @@ spec:
   install:
     strategy: deployment
     spec:
-      clusterPermissions:
-      - serviceAccountName: managed-velero-operator
-        # Rules will be added here by the generate-operator-bundle.py script.
-      permissions:
-      - serviceAccountName: managed-velero-operator
-        # Rules will be added here by the generate-operator-bundle.py script.
-      deployments:
```
Comment (cblecker) [inline config/templates/csv-template.yaml:45]: deployments was removed. Is that intentional?

**Review by cblecker — COMMENTED** [review]
/lgtm
/approve
/hold

One question. If it's safe, you can remove the hold.

File: config/templates/csv-template.yaml (hunk 1 above)
Comment (jharrington22) [reply config/templates/csv-template.yaml:45]: It gets added again [here](https://github.com/openshift/managed-velero-operator/pull/116/files#diff-e12cb709d3181a1ca667b183451c41bbb410dd055344a17d1c0bc981430fde16R288-R294) from the yaml in the deploy dir.

Comment (2uasimojo) [reply config/templates/csv-template.yaml:45]: Yes, [this](https://github.com/openshift/managed-velero-operator/blob/f6939d65884c5b77b61b3830801614b9428d34c0/boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py#L289-L294) will vivify it correctly.

**Pull request merged**